### PR TITLE
Tests repaired - now they're using mocking, and some connector's meth…

### PR DESCRIPTION
**Pull request by pgruszczynski93**
…ods now returns status codes.

### CODE REVIEW

**Comment by Sekobb** [comment]
2nd commit - Small fixes for templates and views

File: PZ/WebClient/tests.py [hunk 1]
```diff
@@ -46,58 +79,65 @@ def test_get_payload(self):
...
-
-    def test_get_measurement(self):
-        print("Tests does get_measurement returns resource")
-        con = Connector(url_measurements)
-        self.assertTrue(con.get_measurement('http://localhost:8002/measurements/61c7524a-84da-4aef-a14b-38f0ad87e08f'))
-
-    def test_is_get_resource_instance_of_measurements(self):
+        self.assertEquals(404, con.delete_resource(guid))
+
+    # @mock.patch('requests.get', side_effect=mocked_requests_get)
```
Comment (Dorrin93) [inline PZ/WebClient/tests.py]: Remove commented tests

File: PZ/WebClient/tests.py [hunk 2]
```diff
@@ -46,58 +79,65 @@ def test_get_payload(self):
...
-        con = Connector(url_measurements)
-        self.assertIsInstance(con.get_measurement('http://localhost:8002/measurements/61c7524a-84da-4aef-a14b-38f0ad87e08f'), Measurements)
+        connector = Connector(url_measurements)
+        self.assertIsInstance(connector.get_measurement(endpoint), Measurements)
 
-    def test_get_measurement_values(self):
-        print("Tests if get_measurement_values returns proper values - empty list")
-        con = Connector(url_measurements)
-        self.assertTrue(len(con.get_measurement_values('http://localhost:8002/measurements/61c7524a-84da-4aef-a14b-38f0ad87e08f')) == 0)
+    # @mock.patch('requests.get', side_effect=mocked_requests_get)
```
Comment (Dorrin93) [inline PZ/WebClient/tests.py]: Remove

**Review by Dorrin93 — COMMENTED** [review]
I approve changes but this commented tests must be removed

**Review by Dorrin93 — APPROVED** [review]

**Pull request merged**